### PR TITLE
fix bug: it could not convert success while the abi equal armhf

### PR DESCRIPTION
**Pull request by Hanlen**

### CODE REVIEW

File: tools/arm_compiler/CROSSTOOL [hunk 1]
```diff
@@ -3,7 +3,7 @@ minor_version: ""
 default_target_cpu: "armhf"
 
 default_toolchain {
-  cpu: "armhf"
+  cpu: "armeabi-v7a"
```
Comment (lu229) [inline tools/arm_compiler/CROSSTOOL]: @Hanlen It is not a bug, armhf is for arm linux, while armeabi-v7a is for android. Do you have any problems on compiling arm linux?

Comment (lu229) [reply tools/arm_compiler/CROSSTOOL]: I run the command "sh tools/cmake-build-arm-linux-gnueabihf-full.sh", and can not reporduct this problem. I noticed that this error was not caused by cmake but the bazel. so what is you bazel command? or the python command that invoke the bazel?

Comment (lu229) [reply tools/arm_compiler/CROSSTOOL]: This is my baze command which compile the arm linux:
python tools/converter.py convert --config=${CONF_FILE}  --target_socs=RK3399 --model_graph_format=file --model_data_format=file --cl_mem_type=buffer --target_abi=armhf --device_yml=${DEVICE_CONF_FILE}
The key parameter is device_yml, which represents the information of arm linux device, you can reference: https://mace.readthedocs.io/en/latest/user_guide/advanced_usage.html#run-you-model-on-the-embedded-device-arm-linux

Comment (Hanlen) [reply tools/arm_compiler/CROSSTOOL]: Thanks for your reply.
Your command works well, but I want it to output as a static lib file.
I run it like this "python tools/converter.py convert --config=${CONF_FILE} --target_abi=armhf  " 
Here is a part of  the config file :
image

Comment (lu229) [reply tools/arm_compiler/CROSSTOOL]: MACE can not compile a model to a static lib, if you means the static lib include only MACE engine, the command "sh tools/cmake-build-arm-linux-gnueabihf-full.sh" will meet your needs.

**Pull request closed without merging**